src/pages/search/advanced_search/CategoryFilterDialog.tsx

```diff
@@ -35,7 +35,12 @@ export const CategoryFilterDialog = ({ open, currentCategories, closeDialog }: C
     <Dialog open={open} onClose={closeDialog} maxWidth="md" fullWidth>
       <DialogTitle>{t('search.select_one_or_more_categories')}</DialogTitle>
       <DialogContent>
-        <CategorySelector selectedCategories={selectedCategories} onCategoryClick={onSelectType} />
+        <CategorySelector
+          canSelectAllNviCategories={true}
```

Comment:
for bolske props holder det å oppgi navnet for å sende inn en true verdi.
i mange tilfeller bruker vi å sette bolske props som optional, med `false` som default verdi, sånn at fraværet av propen betyr false, og navnet betyr true. Da slipper man å ha både opt-out og opt-in på en måte. [Her](https://github.com/BIBSYSDEV/NVA-Frontend/blob/0e3ee203d8067c25e5cf3e179f0d9c4b2732076a/src/components/ConfirmDialog.tsx#L25) er et tilfeldig eksempel

```suggestion
          canSelectAllNviCategories
```

Reply:
Fjernet den helt i stedet.